src/containers/Cfe/selectors.ts

```diff
@@ -30,18 +31,58 @@ export interface VolumeViewerProps {
         translation: [number, number, number];
         rotation: [number, number, number];
     };
+    metadata?: { [key: string]: string | number | null };
     onControlPanelToggle?(collapsed: boolean): void;
 }
 
+function formatFeatureValue(featureValue: number | null, featureDef: MeasuredFeatureDef): string {
+    const { unit, discrete } = featureDef;
+
+    if (featureValue === null || featureValue === undefined) {
+        return `${featureValue}`;
+    }
+    if (discrete) {
+        return `${featureDef.options[featureValue.toString()]?.name}`;
+    }
+
+    const unitSubstitutes: Record<string, string> = {
+        degrees: "deg",
+    };
+    if (unit && unit !== "unitless" && unit !== "stage") {
```

Comment:
tiny concern at special-casing "stage" here - slippery slope of deciding which unit types are meant to be ignored.  But not blocking this PR, just something to think about down the road.

Reply:
I did put my single other arbitrary decision in a map (replace "degrees" with "deg"), to give it a specific, isolated "arbitrary decision zone" where other subjective formatting choices can be evaluated, added, and removed - it may at least make sense to do the same thing here